src/v8/v8_module.rs

```diff
@@ -52,11 +52,11 @@ pub(crate) extern "C" fn load_module<
     // return value.
     // Users can use this isolate score as if it was a regular isolate scope.
     let isolate_scope = V8IsolateScope::new_dummy(&isolate);
-    let ctx_scope = V8ContextScope {
-        inner_ctx_ref: v8_ctx_ref,
-        exit_on_drop: false,
-        isolate_scope: &isolate_scope,
-    };
+    let ctx_scope = V8ContextScope::new_for_ref(
+        unsafe { NonNull::new_unchecked(v8_ctx_ref) },
+        false,
+        &isolate_scope,
+    );
```

Comment:
I guess we can use `get_current_context_scope` (if we change it to pass `false` as I suggested on the comment above).